assets/models/system/automl_instance_segmentation/spec.yaml

```diff
@@ -0,0 +1,24 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: AutoML-Image-Instance-Segmentation
+path: ./
+properties:
+  datasets: COCO
+  finetune-min-sku-spec: 4|1|28|176
+  finetune-recommended-sku: Standard_NC6s_v3
+  finetuning-tasks: image-object-detection
+  inference-min-sku-spec: 4|0|14|28
+  inference-recommended-sku: Standard_DS3_v2, Standard_D4a_v4, Standard_D4as_v4, Standard_DS4_v2,
+    Standard_D8a_v4, Standard_D8as_v4, Standard_DS5_v2, Standard_D16a_v4, Standard_D16as_v4,
+    Standard_D32a_v4, Standard_D32as_v4, Standard_D48a_v4, Standard_D48as_v4, Standard_D64a_v4,
+    Standard_D64as_v4, Standard_D96a_v4, Standard_D96as_v4, Standard_FX4mds, Standard_F8s_v2,
+    Standard_FX12mds, Standard_F16s_v2, Standard_F32s_v2, Standard_F48s_v2, Standard_F64s_v2,
+    Standard_F72s_v2, Standard_FX24mds, Standard_FX36mds, Standard_FX48mds, Standard_E4s_v3,
+    Standard_E8s_v3, Standard_E16s_v3, Standard_E32s_v3, Standard_E48s_v3, Standard_E64s_v3,
+    Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC8as_T4_v3, Standard_NC12s_v3,
+    Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3, Standard_NC24ads_A100_v4,
+    Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4,
+    Standard_ND40rs_v2
+tags:
+  license: apache-2.0
```

Comment:
Is this license information correct. I see that we have it listed under different license in our azuremlcli repo is different: https://msdata.visualstudio.com/Vienna/_git/AzureMlCli?path=/src/azureml-automl-dnn-vision/azureml/automl/dnn/vision/common/NOTICE&version=GBmaster&line=734&lineEnd=735&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
https://arxiv.org/abs/1703.06870 --> https://github.com/facebookresearch/Detectron/blob/main/LICENSE